metrics/network/iperf3_dockerfile/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Usage: FROM [image name]
+FROM fedora
+
+# Version of the Dockerfile
+LABEL DOCKERFILE_VERSION="1.0"
+
+# Install iperf3
+RUN dnf -y update && \
+    dnf -y install iperf3
+
+CMD ["/bin/bash"]
```

Comment:
why bash and not iperf3 ?

Reply:
@devimc , I replaced the `CMD` for `ENTRYPOINT` with `iperf3`, however, it have issues to run the client for example
```
$  docker run --runtime kata-runtime local-iperf3 'export TMPDIR=/dev/shm && iperf3 -c 172.17.0.2 -u -t 30'
iperf3: parameter error - must either be a client (-c) or server (-s)
```
The main issue seems that it does not accept the `export TMPDIR...` that we need to run the container

Reply:
make sense, thanks @GabyCT

Reply:
Also, sometimes we might want the container to be the server, and sometimes the client, so hard coding the `iperf3` commandline as the `CMD` might not scale out so well.